index.html

```diff
@@ -1,14 +1,23 @@
 <!DOCTYPE html>
-<html>
+<html lang ="en" data-ng-app="radioBip">
   <head>
     <title>WDI Radio</title>
-    <script src="js/app.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.15/angular-ui-router.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.0-beta.2/angular-resource.min.js"></script>
+
+<script src="js/app.js"></script>
+<script src="js/songs.module.js"></script>
+<script src="js/index.controller.js"></script>
+<script src="js/radioBip.factory.js"></script>
```

Comment:
I find it more helpful to include the model name in the file name, or at least something that indicates the functionality of this factory.